common/djangoapps/student/tests/test_models.py

```diff
@@ -142,3 +143,16 @@ def test_upgrade_deadline_for_non_upgradeable_enrollment(self, mode):
         """ The property should return None if an upgrade cannot be upgraded. """
         enrollment = CourseEnrollmentFactory(course_id=self.course.id, mode=mode)
         self.assertIsNone(enrollment.upgrade_deadline)
+
+    @skip_unless_lms
+    def test_upgrade_deadline_instructor_paced(self):
+        """ Foo """
+        course = CourseFactory(self_paced=True)
```

Comment:
Shouldn't this be `self_paced=False` since this test is for instructor-paced?  And, if this test is passing, then there may be an issue with the setup.

Reply:
Yup - good catch! I thought I ran this on my devstack... must not have. Should be fixed now.